js/main.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListener('DOMContentLoaded', function(){
+  if (window.location.search == '?signup=confirmed') {
+    var post = document.querySelector('.header');
+
+    var confirmationEle = document.createElement('div');
+    confirmationEle.className = 'announcement';
+    confirmationEle.setAttribute('role', 'alert');
+    confirmationEle.innerHTML = '<div class="container"><p>Successfully signed up for updates</p></div>';
```

Comment:
I'm not getting this @garrying ! tested in a few browsers. does this work for you? 

I feel like my brain is broken atm, but I think i switched something with the header when I pulled code over

Reply:
OMG! I'm silly. I was actually getting redirected from formspree to http://ournetworks.ca/?signup=confirmed so of course it wasn't showing local bits. Okay. nvm.